src/installation/guides/fde.md

```diff
@@ -233,6 +245,8 @@ a new file at `/etc/dracut.conf.d/10-crypt.conf` with the following line:
 install_items+=" /boot/volume.key /etc/crypttab "
 ```
 
+## Finish installation
```

Comment:
If this must be a header, maybe "Last installation steps" or "Complete system installation"